kafka-ui-react-app/src/components/Connect/Details/Tasks/Tasks.tsx

```diff
@@ -45,7 +47,9 @@ const Tasks: React.FC = () => {
             <td>
               <Tag color={getTagColor(task.status)}>{task.status.state}</Tag>
             </td>
-            <td>{task.status.trace || 'null'}</td>
+            <td>
+              <TraceCell stackTrace={task.status.trace || ''} />
+            </td>
```

Comment:
I would suggest to use TopicMessages approach. So we need to add expandable row with message

Reply:
Fixed